docs/source/molecular_generation.rst

```diff
@@ -11,6 +11,28 @@ Various fingerprint types are available on the Citrine Platform, including Atom
 Each fingerprint type captures different aspects of molecular structure and influences the generated mutations.
 You can access these fingerprint types through the :class:`~citrine.informatics.generative_design.FingerprintType` enum, like `FingerprintType.ECFP4`.
 
+The `structure_exclusions` parameter allows you to control the structural features of mutated molecules.
+It is a sequence of exclusion types corresponding to the types of structural features or elements to exclude from the list of possible mutation steps during the generative design process.
+If a type is present in the sequence, the mutation steps generated by the process will avoid using that feature or element.
+The available structure exclusion options can be found in the :class:`~citrine.informatics.generative_design.StructureExclusion` class.
+
+The available structure exclusion types are:
```

Comment:
Since we're pointing them towards the `StructuredExclusion` class for the available options, we probably shouldn't replicate them here. Otherwise, we'll have two lists to update any time we add or remove one (although I imagine that will be rare).